File: app/src/main/java/com/roy/turbo/launcher/FolderKeyEventListener.java
package com.roy.turbo.launcher;

import android.view.KeyEvent;
import android.view.View;

/**
 * A keyboard listener we set on all the workspace icons.
 */
class FolderKeyEventListener implements View.OnKeyListener {
    public boolean onKey(View v, int keyCode, KeyEvent event) {
        return FocusHelper.handleFolderKeyEvent(v, keyCode, event);
    }
}

File: app/src/main/java/com/roy/turbo/launcher/FolderKeyEventListener.kt
package com.roy.turbo.launcher

import android.view.KeyEvent
import android.view.View

/**
 * A keyboard listener we set on all the workspace icons.
 */
internal class FolderKeyEventListener : View.OnKeyListener {
    override fun onKey(v: View, keyCode: Int, event: KeyEvent): Boolean {
        return FocusHelper.handleFolderKeyEvent(v, keyCode, event)
    }
}
